spec/spec_helper.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require 'pry'
```

Comment:
Minor: Perhaps we are keeping Pry, it's nice to have access to, when fiddling with things?